domlab.js

```diff
@@ -0,0 +1,13 @@
+// f to c cnvertor function
+
+function calc(userInput){
+    userInput = document.getElementsByClassName('userInput').value
+    let c = document.getElementsByClassName('c').value
+    c += userInput - 32 * (5/9)
+    console.log(c)
```

Comment:
You should update the celsius value on the screen with the calculated value, not `console.log()` the value (-5)